packages/compass-indexes/src/components/regular-indexes-table/status-field.tsx

```diff
@@ -81,9 +81,9 @@ const StatusField: React.FunctionComponent<StatusFieldProps> = ({
         </BadgeWithTooltip>
       )}
 
-      {status === 'inprogress' && (
-        <Badge data-testid="index-in-progress" variant={BadgeVariant.Blue}>
-          In Progress
+      {status === 'creating' && (
```

Comment:
We probably want both here, created first while we are "creating" it, then "in progress" one while it is being created, I think the UI right now is built around the idea that there is always some meaningful status we should display for the user (even if "Building..." info is being shown near it)